services/libs/integrations/src/integrations/premium/hubspot/api/batchCreateOrganizations.ts

```diff
@@ -109,45 +116,46 @@ export const batchCreateOrganizations = async (
         { err },
         'Conflict while batch create companies in HubSpot. Trying to resolve the conflicts.',
       )
-      const match = err.response?.data?.message.match(/ID: (\d+)/)
-      const id = match ? match[1] : null
-      if (id) {
-        const organization = await getCompanyById(nangoId, id, organizationMapper, ctx, throttler)
-
-        if (organization) {
-          // exclude found organization from batch payload
-          const createOrganizations = organizations.filter(
-            (o) => !o.website.includes((organization.properties as any).domain),
-          )
-
-          const updateOrganizations = organizations
-            .filter((o) => o.website.includes((organization.properties as any).domain))
-            .map((o) => {
-              o.attributes = {
-                ...o.attributes,
-                sourceId: {
-                  hubspot: id,
-                },
-              }
-              return o
-            })
-
-          await batchUpdateOrganizations(
-            nangoId,
-            updateOrganizations,
-            organizationMapper,
-            ctx,
-            throttler,
-          )
-          return await batchCreateOrganizations(
-            nangoId,
-            createOrganizations,
-            organizationMapper,
-            ctx,
-            throttler,
-          )
-        }
-      }
+      // TODO uros fix this
+      // const match = err.response?.data?.message.match(/ID: (\d+)/)
+      // const id = match ? match[1] : null
+      // if (id) {
+      //   const organization = await getCompanyById(nangoId, id, organizationMapper, ctx, throttler)
+
+      //   if (organization) {
+      //     // exclude found organization from batch payload
+      //     const createOrganizations = organizations.filter(
+      //       (o) => !o.website.includes((organization.properties as any).domain),
+      //     )
+
+      //     const updateOrganizations = organizations
+      //       .filter((o) => o.website.includes((organization.properties as any).domain))
+      //       .map((o) => {
+      //         o.attributes = {
+      //           ...o.attributes,
+      //           sourceId: {
+      //             hubspot: id,
+      //           },
+      //         }
+      //         return o
+      //       })
+
+      //     await batchUpdateOrganizations(
+      //       nangoId,
+      //       updateOrganizations,
+      //       organizationMapper,
+      //       ctx,
+      //       throttler,
+      //     )
+      //     return await batchCreateOrganizations(
+      //       nangoId,
+      //       createOrganizations,
+      //       organizationMapper,
+      //       ctx,
+      //       throttler,
+      //     )
+      //   }
+      // }
```

Comment:
Idea here was if we tried creating an existing organization HubSpot would return a conflict error - We parse the conflicting id for the batch create request and update these conflicting ones instead (removing the conflicting id from the batch create and retrying)
Since HubSpot integration is not used anymore, I'm not sure how relevant it's to try to maintain it with the changing code tbh